src/Event.cs

```diff
@@ -62,6 +62,11 @@ public SubscriptionBuilder<EventContext> OnError(Action<EventContext> callback)
         }
 
         public SubscriptionHandle<EventContext> Subscribe(params string[] querySqls) => new(conn, Applied, Error, querySqls);
+
+        public void SubscribeToAllTables() {
+            string[] querySqls = {"SELECT * FROM *"};
+            new SubscriptionHandle<EventContext>(conn, Applied, Error, querySqls);
```

Comment:
We accept `params` in the API, which allows variadic params as a syntax sugar instead of an array, so you can just do

```suggestion
            new SubscriptionHandle<EventContext>(conn, Applied, Error, "SELECT * FROM *");
```